presto-docs/src/main/sphinx/release/release-0.237.rst

```diff
@@ -0,0 +1,57 @@
+=============
+Release 0.237
+=============
+
+General Changes
+_______________
+* Fix NPE in common sub-expression optimization triggered by ``CASE-WHEN`` expression.
+* Fix compiler failure with function type when common sub-expression optimization is enabled.
+* Improve coordinator RPC performance by removing unused entries from ``TaskStatus``.
+* Add JSON format for ``EXPLAIN`` with type ``LOGICAL`` and ``DISTRIBUTED``.
+* Add functions :func:`array_sum`, :func:`map_normalize` and :func:`set_agg`.
```

Comment:
I believe we also added function array_average. Could you please add it to the list?